diagnostic.js

```diff
@@ -32,7 +41,40 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update(
+   { name: "House Stark" },
+   {
+     name: 'House Stark',
+     motto: 'Winter is Coming',
+     members: ['Ned Stark', 'Arya Stark', 'Sansa Stark']
+   }
+);
+db.houses.update(
+  { name: "House Targaryen"}, {
+    $set:
+      {
+        members: ['Viserys Targaryen', 'Daenerys Targaryen']
+      }
+  }
+);
+db.houses.update(
+  { name: 'House Arryn'},
+  {
+    $set:
+    {
+      members: []
+    }
+  }
+);
+db.houses.update(
+  { name: 'House Arryn'},
+  {
+    $push:
+      {
+        members: 'Jon Arryn'
+      }
+  }
+);
```

Comment:
nice! 1 quick thing: for Arryn, you could've included members in your `$set`, like this:
```js
db.houses.update(
  {name: 'House Arryn'},
  {$set: {members: ['Jon Arryn']}}
)
```